ctapipe/io/eventfilereader.py

```diff
@@ -149,215 +98,202 @@ def check_file_compatibility(file_path):
 
     @property
     @abstractmethod
-    def num_events(self):
+    def camera(self):
```

Comment:
This confuses me. Why is this a property of the EventFIleReader? A File might contain array events with very different Cameras like it's the case with the current MCs.